frontend/src/components/dialogs/GDialog.vue

```diff
@@ -86,30 +86,34 @@ SPDX-License-Identifier: Apache-2.0
         </v-btn>
         <v-tooltip
           location="top"
-          :disabled="valid"
+          :disabled="!notConfirmed || hasVisibleErrors || !valid"
         >
           <template #activator="{ props }">
             <div v-bind="props">
-              <v-btn
+              <g-vuelidate-button
+                :v="v$"
```

Comment:
I don't think this is a good idea to pass the full vuelidate object down into another component.

Reply:
removed component as discussed